develop/parachain-devs/parachain-launch/chain-spec.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Chain Specification
+description: Describes the role of the chain specification in a network, how to specify the chain specification to use when starting a node, and how to customize and distribute chain specifications.
+---
+
+A _chain specification_ is the collection of information that describes a Polkadot SDK-based network. When starting a node, a chain specification is a crucial parameter, providing the genesis configurations, bootnodes, and other parameters relating to that particular network. It identifies the network that a blockchain node connects to, the other nodes that it initially communicates with, and the initial state that nodes must agree on to produce blocks.
+
+The chain specification is defined using the [`ChainSpec` struct](https://paritytech.github.io/polkadot-sdk/master/sc_chain_spec/struct.GenericChainSpec.html){target=_blank}.
```

Comment:
nit: Maybe it would be better to describe a json that describes chain-spec?
Not sure if divin into rust structures makes sense.

But this depends on who is the target audience of this spec.
There is a description of fields here:
https://paritytech.github.io/polkadot-sdk/master/sc_chain_spec/index.html#intro-chain-specification

What reminds me that we need a json vaidation scheme for chain spec file (of course out of scope of this PR).

Reply:
+1. I think it is better to refer to the Polkadot SDK docs instead of the Rust struct